tools/KoreBuild.Console/Commands/DockerFiles/winservercore.dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM microsoft/aspnet:4.6.2
+
+WORKDIR c:\\repo
```

Comment:
Move `WORKDIR` after `RUN` to improve caching, so if you decide to change the `WORKDIR` you don't need to re-execute the `RUN` command.